src/network/networkctl.c

```diff
@@ -3216,6 +3328,8 @@ static int help(void) {
                "  reload                 Reload .network and .netdev files\n"
                "  edit                   Edit network configuration files\n"
                "  cat                    Show network configuration files\n"
+               "  edit-link              Edit network configuration files for the given link\n"
+               "  cat-link               Show network configuration files for the given link\n"
```

Comment:
so these take device names? 

This is weird. Both .link and .network files may apply to a bunch of network interfaces. It's weird to make the edit logic for one take file names and for the other take netdev names.

it appears to me we should maybe in all cases assume the argument is an fname, but if it starts with `@` then interpret it as interface name instead, and then first look up file currently used on that interface.